docker-compose.yml

```diff
@@ -147,26 +147,28 @@ services:
     volumes:
       - minio_data:/data
     environment:
-      MINIO_ACCESS_KEY: minio
-      MINIO_SECRET_KEY: coopcycle
-      MINIO_REGION: fr-fr
+      - MINIO_ACCESS_KEY=${S3_CREDENTIALS_KEY}
+      - MINIO_SECRET_KEY=${S3_CREDENTIALS_SECRET}
+      - MINIO_REGION=${S3_REGION}
 
   # https://github.com/minio/minio/issues/4882
   # https://docs.min.io/docs/minio-client-complete-guide#policy
   minio_client:
     image: minio/mc
     depends_on:
       - minio
+    environment:
+      - S3_REGION
     entrypoint: >
       /bin/sh -c "
       /usr/bin/mc config host add coopcycle http://minio:9000 minio coopcycle;
```

Comment:
Not sure how to build this config line, help needed

Reply:
Just sent an update, [minio docs](https://docs.min.io/docs/minio-client-complete-guide.html) helped me out